frontend/src/components/smart/CharacterList.vue

```diff
@@ -1,30 +1,40 @@
 <template>
   <div>
-    <div class="filters row mt-2" v-if="showFilters">
-      <div class="col-sm-6 col-md-4 col-lg-3 mb-3">
+    <div class="filters row mt-2 pl-2" v-if="showFilters" @change="saveFilters()">
```

Comment:
does @change fire on a div?

Reply:
It dosen't fire directly on div but, every events bubble/propagate to the parents, so to prevent listen the change event on each child we can just listen to the parent

Reply:
I see. I don't like that solution only because it's a bit confusing. But, if it works, it works. I'm currently working on some API code but I'll merge this asap.

Reply:
Ty, do not hesitate to tell me if you want me to edit anything.